code/data/multi_country/convert-policies-raw-to-interim.py

```diff
@@ -0,0 +1,150 @@
+import pandas as pd
+import numpy as np
+import src.utils as cutil
+import json
+import operator
+import argparse
+
+
+def policy_conversion_parser():
```

Comment:
curious why you put this in a dictionary. I think it works fine! Just hadn't seen this approach before so am curious :)

Reply:
a function? no real reason, i just thought it's a little easier on the eyes to group those things in an indent

Reply:
yup - a function. Cool! agreed.